CMakeLists.txt

```diff
@@ -51,9 +51,9 @@ endif()
 if(ARM_TARGET MATCHES "^STM32" AND USE_UART0)
   message(FATAL_ERROR "UART0 is not supported for STM32 boards")
 endif()
-# We don't support UART1 on STM32F4 Discovery kit
-if(ARM_TARGET STREQUAL "STM32F407" AND USE_UART1)
-  message(FATAL_ERROR "UART1 is not supported for STM32F407")
+# We don't support UART1 on STM32F4 Discovery kit and STM32L4R5
+if((ARM_TARGET STREQUAL "STM32F407" OR ARM_TARGET STREQUAL "STM32L4R5") AND USE_UART1)
+  message(FATAL_ERROR "UART1 is not supported for STM32F407 and STM32L4R5")
```

Comment:
Sorry, I think I broke this with https://github.com/iml130/iree-bare-metal-arm/commit/249a4800997d1d604fefa7504713df120600ddd8. Can you please rebase upon the latest main?